data/food/export_agb/export_builder.py

```diff
@@ -109,13 +128,22 @@ def compute_new_processes(activities, ingredients_base):
             for d in list(csv.DictReader(f, dialect="unix"))
         ]
 
-    # Parse the ingredients_base.json, which may contain complex ingredients to add/compute
+    # Parse the ingredients.json, which may contain complex ingredients to add/compute
```

Comment:
Are we not loading `ingredients_base.json` anymore? If not, then maybe rename the global variable on the next line?

Reply:
ingredients_base.json is still parsed and the source of the ingredients.
This comment was changed during a too quick variable renaming...